crates/orderbook/openapi.yml

```
@@ -540,32 +530,113 @@ components:
+        sellAmount:
+          description: see `OrderParameters::sellAmount`
+          allOf:
+            - $ref: "#/components/schemas/TokenAmount"
+        buyAmount:
+          description: see `OrderParameters::buyAmount`
+          allOf:
+            - $ref: "#/components/schemas/TokenAmount"
+        validTo:
+          description: see `OrderParameters::validTo`
+          type: integer
+        feeAmount:
+          description: see `OrderParameters::feeAmount`
+          allOf:
+            - $ref: "#/components/schemas/TokenAmount"
+        kind:
+          description: see `OrderParameters::kind`
+          allOf:
+            - $ref: "#/components/schemas/OrderKind"
+        partiallyFillable:
+          description: see `OrderParameters::partiallyFillable`
+          type: boolean
+        sellTokenBalance:
+          description: see `OrderParameters::sellTokenBalance`
+          allOf:
+            - $ref: "#/components/schemas/SellTokenSource"
+          default: "erc20"
+        buyTokenBalance:
+          description: see `OrderParameters::buyTokenBalance`
+          allOf:
+            - $ref: "#/components/schemas/BuyTokenDestination"
+          default: "erc20"
+        signingScheme:
+          $ref: "#/components/schemas/SigningScheme"
+        signature:
+          $ref: "#/components/schemas/Signature"
+        from:
+          description: |
+            If set, the backend enforces that this address matches what is decoded as the signer of
+            the signature. This helps catch errors with invalid signature encodings as the backend
+            might otherwise silently work with an unexpected address that for example does not have
+            any balance.
+          allOf:
+            - $ref: "#/components/schemas/Address"
+          nullable: true
+        quoteId:
+          description: |
+            Orders can optionally include a quote ID. This way the order can be linked to a quote
+            and enable providing more metadata when analyzing order slippage.
+          type: integer
+          nullable: true
+        appData:
+          description: |
+            This field comes in two forms for backward compatibility. The first form (app data hash) will eventually stop being accepted.
+          anyOf:
+            - $ref: "#/components/schemas/AppDataHash"
+            - title: Full App Data
               description: |
-                Orders can optionally include a quote ID. This way the order can be linked to a quote
-                and enable providing more metadata when analyzing order slippage.
-              type: integer
-              nullable: true
-          required:
-            - signingScheme
-            - signature
+                Short:
+
+                If you do not care about app data, set this field to `\"{}\"` and make sure that the order you signed for this request had its app data field set to `0xb48d38f93eaa084033fc5970bf96e559c33c4cdc07d889ab00b4d63f9590739d`.
```

I think you meant to say this:

```suggestion
                If you do not care about app data, set this field to `\"{}\"` and make sure that the order you signed for this request had its appDataHash field set to `0xb48d38f93eaa084033fc5970bf96e559c33c4cdc07d889ab00b4d63f9590739d`.
```

---

It depends what the library the api user is using calls it. In our official library it is called "app data". https://github.com/cowprotocol/contracts/blob/5c25837f585d79dc5450fb80f66c41f596c3198d/src/ts/order.ts#L49